Symfony/CS/Tests/Fixer/AbstractFixerTestBase.php

```diff
@@ -69,26 +69,24 @@ protected function makeTest($expected, $input = null, \SplFileInfo $file = null,
 
         if ($fileIsSupported) {
             $fixResult = $fixer->fix($file, $tokens);
-            $this->assertNull($fixResult, '->fix method should return null.');
+            $this->assertNull($fixResult, '->fix method must return null.');
         }
 
+        $this->assertFalse($tokens->isChanged(), 'Tokens collection built on expected code must not be marked as changed after fixing.');
         $this->assertSame($expected, $tokens->generateCode(), 'Code build on expected code must not change.');
-        $this->assertFalse($tokens->isChanged(), 'Tokens collection built on expected code should not be marked as changed after fixing.');
     }
 
-    private function assertTokens(Tokens $expectedTokens, Tokens $tokens)
+    private function assertTokens(Tokens $expectedTokens, Tokens $inputTokens)
     {
-        foreach ($expectedTokens as $index => $expectedToken) {
-            $token = $tokens[$index];
+        $this->assertEquals($expectedTokens->count(), $inputTokens->count(), 'The collection must have the same length than the expected one.');
```

Comment:
checking the length at the end was actually expected, to have a better chance to see the difference between collections, to help debugging (if you only know that the length is different, it is hard to debug this)